smdebug/tensorflow/keras.py

```diff
@@ -107,11 +107,14 @@ def _get_matching_collections(
                 continue
 
             if match_inc(ts_name, current_coll.include_regex):
-                # In TF 2.x, we can't put tensors in a set/dictionary as tensor.__hash__()
+                # In TF 2.x eager mode, we can't put tensors in a set/dictionary as tensor.__hash__()
                 # is no longer available. tensor.experimental_ref() returns a hashable reference
                 # object to this Tensor.
                 check_tensor = tensor
```

Comment:
do we need this assignment. 

We can update tensor in line 118 and use tensor in line 119

Reply:
done in the latest commit